int64 velPriority
float64 velData
bool velRelease
float64 disData
